Exam/CT/src/exercice3/model/Board.java

```diff
@@ -45,6 +45,48 @@ public Board(int nbRows, int nbColumns) {
 		}
 		return c;
 	}
+	
+	public int computeNeighboringBomb(int row, int col) {
+		int c = 0;
+		if (row > 0) {
+			if (col > 0)
+				if(cells[row - 1][col - 1].isBomb == true) {
+					c = c + 1;
+				}
+			if(cells[row - 1][col].isBomb == true) {
+				c = c +1;
+			}
+			if (col < nbColumns - 1)
+				if(cells[row - 1][col + 1].isBomb == true) {
+					c = c + 1;
+				}
+		}
+		if (col > 0)
+			if(cells[row][col - 1].isBomb == true) {
+				c = c + 1 ;
+			}
+		if (col < nbColumns - 1)
+			if(cells[row][col + 1].isBomb == true) {
+				c = c + 1;
+			}
+
+		if (row < nbRows - 1) {
+			if (col > 0) {
+				if(cells[row + 1][col - 1].isBomb == true) {
+					c = c + 1;
+				}
+			}
+			if(cells[row + 1][col].isBomb == true) {
+				c = c + 1;
+			}
+			if (col < nbColumns - 1) {
+				if(cells[row + 1][col + 1].isBomb == true) {
+					c = c+1;
+				}
+			}
+		}
+		return c;
```

Comment:
So you've computed the number of neighboring bombs. But what should you do with that...